lighthouse-core/gather/fetcher.js

```diff
@@ -14,13 +14,15 @@
 /* global document */
 
 const log = require('lighthouse-logger');
+const {getBrowserVersion} = require('./driver/environment.js');
 
 class Fetcher {
   /**
-   * @param {import('./driver.js')} driver
+   * @param {LH.Gatherer.FRTransitionalDriver} driver
    */
   constructor(driver) {
-    this.driver = driver;
+    this.session = driver.defaultSession;
```

Comment:
FWIW could avoid the sticky issue of the other properties being ready/demanding a particular initialization order by saving `this._driver = driver` instead, not a big deal though